tests/Doctrine/Tests/DBAL/Functional/DataAccessTest.php

```diff
@@ -682,14 +680,12 @@ public function testBitComparisonExpressionSupport(): void
             ]);
         }
 
-        $sql[] = 'SELECT ';
-        $sql[] = 'test_int, ';
-        $sql[] = 'test_string, ';
-        $sql[] = $platform->getBitOrComparisonExpression('test_int', 2) . ' AS bit_or, ';
-        $sql[] = $platform->getBitAndComparisonExpression('test_int', 2) . ' AS bit_and ';
-        $sql[] = 'FROM fetch_table';
+        $sql = 'SELECT test_int, test_string'
+            . ', ' . $platform->getBitOrComparisonExpression('test_int', 2) . ' AS bit_or'
+            . ', ' . $platform->getBitAndComparisonExpression('test_int', 2) . ' AS bit_and'
+            . ' FROM fetch_table';
```

Comment:
```suggestion
        $sql = sprintf(
        <<<'SQL'
        SELECT
            test_int,
            test_string,
            %s AS bit_or,
           %s as bit_and
        FROM fetch_table
SQL
,
            $platform->getBitOrComparisonExpression('test_int', 2), 
            $platform->getBitAndComparisonExpression('test_int', 2)
        );
```
modulo the cs issues of course